simple-blocks/main.ts

```diff
@@ -0,0 +1,47 @@
+namespace simplified {
+
+    /**
+     * Allows your sprite to jump only if it's currently
+     * on a floor or without velocity.
+     * Expand the + to add jumping power.
+     */
+    //% group=Sprites
+    //% color="#3b6fe9"
+    //% blockId=gravity_jump
+    //% block="make $thisSprite=variables_get(mySprite) gravity jump || with power $velo"
+    //% velo.defl=-200
+    //% help=github:simple-blocks/docs/gravity-jump
+    export function gravity_jump(thisSprite: Sprite, velo?: number) {
+        if (!velo) {
```

Comment:
!velo will also trigger if the velocity is equal to 0. Better to do a check like this:
```suggestion
        if (velo == undefined) {
```